rsmtool/notebooks/header.ipynb

```diff
@@ -169,6 +169,11 @@
     "groups_desc = environ_config.get('GROUPS_FOR_DESCRIPTIVES') \n",
     "groups_eval = environ_config.get('GROUPS_FOR_EVALUATIONS') \n",
     "\n",
+    "# min number of n for group to be displayed in the report\n",
+    "min_n_per_group = environ_config.get('MIN_N_PER_GROUP', {})\n",
+    "if not min_n_per_group:\n",
```

Comment:
I think you don't need this `if` statement anymore?

Reply:
Yikes!